public/app/plugins/datasource/cloud-monitoring/dashboards/https-loadbalancer-monitoring.json

```diff
@@ -1131,13 +850,10 @@
       }
     ]
   },
-  "time": {
-    "from": "now-3h",
-    "to": "now"
-  },
+  "time": { "from": "now-24h", "to": "now" },
   "timepicker": {},
   "timezone": "",
-  "title": "HTTP/S GCP Load Balancer Monitoring",
-  "uid": "WO1Za5FMl",
-  "version": 3
+  "title": "HTTP/S Load Balancer Monitoring",
```

Comment:
I think you can not change the title because it is used for forming the dashboard slug:
https://github.com/grafana/grafana/blob/dd2d206d99f7ea8341c1f7242d3026205539aac3/pkg/models/dashboards.go#L275-L279

which in turn is used for finding existing dashboards:
https://github.com/grafana/grafana/blob/c2cad26ca980dcca932dc8308dd545d321eb6246/pkg/plugins/dashboards.go#L65

So if one has imported this dashboard before this PR it will not be found and the `api/plugins/stackdriver/dashboards` response contains a record like:

```
    {
        "pluginId": "",
        "title": "",
        "imported": false,
        "importedUri": "",
        "importedUrl": "",
        "slug": "http-s-gcp-load-balancer-monitoring",
        "dashboardId": 430,
        "folderId": 0,
        "importedRevision": 0,
        "revision": 0,
        "description": "",
        "path": "",
        "removed": true
    }
```

instead of:

```
    {
        "pluginId": "stackdriver",
        "title": "HTTP/S GCP Load Balancer Monitoring",
        "imported": true,
        "importedUri": "db/http-s-gcp-load-balancer-monitoring",
        "importedUrl": "/d/WO1Za5FMl/http-s-gcp-load-balancer-monitoring",
        "slug": "",
        "dashboardId": 430,
        "folderId": 0,
        "importedRevision": 1,
        "revision": 1,
        "description": "",
        "path": "dashboards/https-loadbalancer-monitoring.json",
        "removed": false
    },
```

As a result the tab of the cloud monitoring dashboards contains an empty line at the bottom.
<img width="1552" alt="Screenshot 2020-12-21 at 11 13 37 AM" src="https://user-images.githubusercontent.com/1632407/102760160-c8934600-437d-11eb-8888-ae9fba5112e5.png">